packages/@sanity/dashboard/src/widgets/sanity-tutorials/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import sanityClient from '@sanity/client'
+import imageUrlBuilder from '@sanity/image-url'
+import styles from './index.css'
+import Tutorial from './Tutorial'
+import {get} from 'lodash'
+import {distanceInWords} from 'date-fns'
+const client = sanityClient({
+  projectId: '3do82whm',
+  dataset: 'production',
+  useCdn: true
+})
+
+const builder = imageUrlBuilder(client)
+
+const query = `
+  *[_id == 'dashboardFeed-v1'] {
+    items[]-> {
+      _id,
+      title,
+      poster,
+      youtubeURL,
+      "presenter": authors[0]-> {name, mugshot, bio},
+      guideOrTutorial-> {
+        title,
+        slug,
+        "presenter": authors[0]-> {name, mugshot, bio},
+        _createdAt
+      }
+    }
+  }[0]
+`
+
+function createUrl(slug) {
+  return `https://www.sanity.io/guide/${slug.current}`
+}
+
+class SanityTutorials extends React.Component {
+  state = {
+    feedItems: []
+  }
+
+  componentDidMount() {
```

Comment:
You will want to use `client.observable.fetch` here and store the subscription, then unsubscribe in `componentWillUnmount`